### ISSUE TITLE
Command to add the webhook to a repository?

### ISSUE THREAD

**Issue by terrellt** [atmos/hubot-deploy#55]
Is there a plugin for this already? It'd be nice if the bot could do it all without having to navigate per project to the webhooks.

**Comment by atmos** [comment]
Yeah, I think that'd be great. Ideally iterate over all the repos and attach an org hook for easier maintenance, and a repo hook for all user owned repos. `/deploy:hook sync` or something similar.
